hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -228,7 +228,8 @@ public void testAllBucketOperations() throws Exception {
     Assert.assertEquals(bucketName, ozoneBucket.getName());
     Assert.assertTrue(ozoneBucket.getVersioning());
     Assert.assertEquals(StorageType.DISK, ozoneBucket.getStorageType());
-    Assert.assertTrue(ozoneBucket.getCreationTime() <= Time.now());
+    Assert.assertTrue(ozoneBucket.getCreationTime().compareTo(
+        Instant.now()) <= 0);
```

Comment:
I think using `isAfter`/`isBefore` would be easier to understand at a glance.

```
assertFalse(ozoneBucket.getCreationTime().isAfter(Instant.now()))
```

(I hope I got the conversion right).

Similarly in all other places where `compareTo()` is used.

Reply:
Good point. Fixed.